Library/ViewModels/MessagesViewModel.swift

```diff
@@ -168,11 +168,17 @@ public final class MessagesViewModel: MessagesViewModelType, MessagesViewModelIn
       }
     )
 
-    self.replyButtonIsEnabled = Signal.merge(
-      self.viewDidLoadProperty.signal.mapConst(false),
+    self.participantPreviouslyBlocked = self.project
+      .map { $0.creator.isBlocked }
```

Comment:
Nit, kind of out of scope: It feels risky to me to check creator here instead of participant, since theoretically the creator could be the person doing the messaging. Right now, I don't think that's possible (since those messages don't show in the inbox on mobile), but that could (and maybe should) change in the future.